src/pages/TCY/components/TCYCta.tsx

```diff
@@ -0,0 +1,34 @@
+import { Alert, AlertDescription, AlertTitle, Button, Stack } from '@chakra-ui/react'
+import { tcyAssetId } from '@shapeshiftoss/caip'
+import { useCallback } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useNavigate } from 'react-router'
+
+import { AssetIcon } from '@/components/AssetIcon'
+import { RawText } from '@/components/Text'
+
+const margin = { base: 4, md: 0 }
+
+export const TCYCta = () => {
+  const navigate = useNavigate()
+  const translate = useTranslate()
+
+  const handleClick = useCallback(() => {
+    navigate('/tcy')
+  }, [navigate])
+
+  return (
+    <Alert status='info' m={margin} width='auto'>
+      <AssetIcon assetId={tcyAssetId} />
+      <Stack spacing={0} ml={4}>
+        <AlertTitle>{translate('TCY.cta.title')}</AlertTitle>
+        <AlertDescription>
+          <RawText color='text.subtle'>{translate('TCY.cta.body')}</RawText>
+        </AlertDescription>
+      </Stack>
+      <Button ml='auto' flexShrink={0} onClick={handleClick}>
+        {translate('TCY.cta.button')}
```

Comment:
UX q: Since we're fetching claims in `<TCYNavIndicator />`, do we want to do the same here, and make this "You have claims available" when users do have claims?

I can see some value in keeping the current copy if the currently connected wallet doesn't have any, assuming the intent is to have them switch to a wallet that has some.

Reply:
@reallybeard what's the final decision here sir, we're only showing this if claims are available right? 🙏🏽